packages/ckeditor5-image/src/image/utils.js

```diff
@@ -65,7 +65,18 @@ export function getSelectedImageWidget( selection ) {
  * @returns {Boolean}
  */
 export function isImage( modelElement ) {
-	return !!modelElement && modelElement.is( 'element', 'image' );
+	return isElementOfType( modelElement, 'image' );
+}
+
+/**
+ * Checks if the provided model element is of a given type.
+ *
+ * @param {module:engine/model/element~Element} modelElement
+ * @param {String} type
+ * @returns {Boolean}
+ */
+export function isElementOfType( modelElement, type ) {
```

Comment:
This function should be named `isElementOfName` but I'm not sure if this function is needed at all.